src/tools/emacs.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Emacs
+description: Use Dart on Emacs and its flavors
+toc: false
+---
+
+Emacs has a few major and minor modes built for Dart and Flutter development.
+
+All the steps are identical for enabling Flutter since
+[Dart-Snippets][dart-snippets] contain Flutter snippets too.
+[Flutter.el][flutter-mode] only allows you to run Flutter commands inside your Emacs.
+
+* [Dart-Mode][dart-mode] is the main mode for development.
+* [Dart-Snippets][dart-snippets] enables [Visual Studio Code](/tools/vs-code)
+  like snippets and more.
+* [LSP-Mode][lsp-mode] adds IDE-like features to Dart mode.
+
+{% comment %}
+
+Add a vanilla Emacs config that has all above + hooks and stuff.
+
+{% endcomment %}
+  
+To enable and configure all above in [Doom Emacs][doom-emacs], you simply
```

Comment:
the word "simply" here should probably be removed. If something is simple it'll be obvious to the reader that it's simple, if it's not then using the word "simply" essentially just insults the reader's competence.

Reply:
haha, thanks, that was a simple mistake.
My bad I'm not native... It will be removed in the upcoming commits.